### ISSUE TITLE
a11y heading structure: project modules tiles

### ISSUE THREAD

**Issue by khamui** [liqd/adhocracy-plus#1870]
**URL:** 
**user:** 
**expected behaviour:** heading hierarchy should be h1, h2, h3, h4 etc.
**behaviour:** heading h1 exists, but module-tile titles are h3, h2 is missing
**important screensize:**
**device & browser:** 
**Comment/Question:** 
Example see below "noinoi" is h1 and "Geschäftsord...." headings are h3. Missing h2 on this page.
Screenshot?
<img width="1654" alt="Bildschirmfoto 2022-05-30 um 13 13 35" src="https://user-images.githubusercontent.com/52801593/170990388-e30bd50d-d167-44b1-8f2d-b4e5c73ec728.png">

**Comment by phillimorland** [comment]
It would make sense to have a title here, we could also add a sr only title, @mcastro-lqd design wise what do you think works here? should the tiles have a title explaining what they do?

**Comment by mcastro-lqd** [comment]
@phillimorland @khamui I thought this would happen because of Multimodule projects, but its not the case (see screenshot), wouldn't it make more sense like that?  

Single: h1 - Main Project Title ➡️ h2 - Module Titles
Multimodule: h1 - Main project Title ➡️ h2 - Online Participation ➡️ h3 - Module Titles


<img width="1268" alt="Screenshot 2022-06-09 at 17 03 41" src="https://user-images.githubusercontent.com/59610786/172879611-eb4de626-c71f-4a60-874c-fd24e8966c3a.png">

**Comment by phillimorland** [comment]
@mcastro-lqd just to confirm for multi-modules without the timeline (where modules all run at the same time) shall we add a h2 'Online Participation' heading?

**Comment by mcastro-lqd** [comment]
@phillimorland I think it makes sense right? Since they can switch to a previous one
